native/src/main/java/io/ballerina/stdlib/http/transport/contractimpl/sender/http2/Http2ConnectionManager.java

```diff
@@ -165,6 +165,11 @@ void markClientChannelAsStale(HttpRoute httpRoute, Http2ClientChannel http2Clien
         http2StaleClientChannels.add(http2ClientChannel);
     }
 
+    @SuppressWarnings("java:S899")
```

Comment:
Rather than suppressing the warning for ignoring the return value, we can capture the return value and log a warning right? Just like #L199 (we can also replace the code in #L199 with this function)

Reply:
Updated